### PR TITLE
fix(ubuntu18): since this version is EOL, we remove it

### PR DESCRIPTION
**Pull request by fgelcer**
apparently seems like GCE will remove the images
related to this version, as it got EOL.
we disabled the triggers, and then the jobs,
now are must remove the files, so current and
future releases won't create these jobs anymore.

there was an enterprise feature test that used
Ubuntu18, and it was moved to use now Ubuntu22
instead.
the old jobs will be disabled, and the relevant
new jobs created instead.

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fgelcer** [comment]
new jobs, replacing the equivalent that used Ubuntu 18:

https://jenkins.scylladb.com/job/scylla-enterprise/job/rolling-upgrade/job/rolling-upgrade-ubuntu22.04-ldap-authorization-test/
https://jenkins.scylladb.com/job/enterprise-2023.1/job/rolling-upgrade/job/rolling-upgrade-ubuntu22.04-ldap-authorization-test/
https://jenkins.scylladb.com/job/enterprise-2022.2/job/rolling-upgrade/job/rolling-upgrade-ubuntu22.04-ldap-authorization-test/
https://jenkins.scylladb.com/job/enterprise-2022.1/job/rolling-upgrade/job/rolling-upgrade-ubuntu22.04-ldap-authorization-test/
https://jenkins.scylladb.com/job/enterprise-2021.1/job/rolling-upgrade/job/rolling-upgrade-ubuntu22.04-ldap-authorization-test/
https://jenkins.scylladb.com/job/scylla-enterprise/job/SCT_Enterprise_Features/job/LDAP/job/rolling-upgrade-ubuntu22.04-ldap-authorization-test/
https://jenkins.scylladb.com/job/enterprise-2023.1/job/SCT_Enterprise_Features/job/LDAP/job/rolling-upgrade-ubuntu22.04-ldap-authorization-test/
https://jenkins.scylladb.com/job/enterprise-2021.1/job/SCT_Enterprise_Features/job/LDAP/job/rolling-upgrade-ubuntu22.04-test-ldap-authorization/

**Review by fruch — APPROVED** [review]

**Review by Mark-Gurevich — APPROVED** [review]
LGTM

**Review by yaronkaikov — APPROVED** [review]
LGTM

File: docs/artifacts_test.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ hydra run-test artifacts_test --backend aws --config test-cases/artifacts/amazon
 
 ## Ubuntu 18.04 LTS (bionic)
```
Comment (yarongilor) [inline docs/artifacts_test.md:99]: should be switched to 22.04 as well

**Review by yarongilor — COMMENTED** [review]
@fgelcer `Ubuntu 18.04 LTS (bionic)` is not modified in doc.

**Pull request merged**